scraper.rb

```diff
@@ -44,7 +44,7 @@ def datefrom(date)
     term: 51,
     source: mp_url,
   }
-  data[:photo].prepend(base_url) unless data[:photo].nil? or data[:photo].empty?
+  data[:photo] = URI.join(base_url, data[:photo]).to_s unless data[:photo].to_s.empty?
```

Comment:
There's a subtle potential bug here if the URL of a person's page ends up being at a different host from `base_url` for some reason. It's unlikely to happen, especially with the way it's being built up in this scraper. But as a general habit, it's worth defaulting to joining with `mp_url` rather than `base_url`.